tests/nnx/bridge/wrappers_test.py

```diff
@@ -250,7 +250,9 @@ def test_nnx_to_linen(self):
     assert y.shape == (1, 64)
     np.testing.assert_allclose(y, x @ variables['params']['kernel'])
     assert 'nnx' in variables
-    assert isinstance(variables['nnx']['graphdef'], nnx.GraphDef)
+    assert isinstance(
+      variables['nnx']['graphdef'], nnx.graph.NodeDef | nnx.graph.NodeRef
```

Comment:
Should we have a type annotation for `nnx.graph.NodeDef | nnx.graph.NodeRef`?

Reply:
`GraphDef` is now defined as `GraphDef = tp.Union[NodeDef[Node], NodeRef[Node]]` but `isinstance` doesn't like it because its Generic.